### PR TITLE
Create invite_user.html and refact how to send email

### PR DESCRIPTION
**Pull request by raonismaneoto**
<p><b>Feature/Bug description:</b>
Create a html file to be sent by email when an user is invited and refactor how to send emails.
</p>

<p><b>Solution:</b>
I've created an hierarchy, send_email_hierarchy, that handles, with polimorfism, the task of send invitation emails, removal emails etc. 
</p>

<p><b>TODO/FIXME:</b> n/a</p>

### CODE REVIEW

File: backend/templates/invite_user.html [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+                        </span>
+                      </div>
+                    </div>
+                    <div style="height: 50%; background-color: white;">
+                    </div>
+                  </div>
+                </td>
+              </tr>
+            </table>
+
```
Comment (mayzabeel) [inline backend/templates/invite_user.html]: rlb

File: backend/templates/invite_user.html [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+                        <img src="https://firebasestorage.googleapis.com/v0/b/eciis-splab.appspot.com/o/admin_email%2Fcis.png?alt=media&token=ccab7007-97b0-462b-a1d0-38b5da58a2ad" style="width: 100px; margin-top: 5px; margin-left: 20px;"/>
+                      </div>
+                      <div>
+                        <img src="https://firebasestorage.googleapis.com/v0/b/eciis-splab.appspot.com/o/admin_email%2Fopasoms.png?alt=media&token=a71b5c27-a317-417e-aa6e-3c744ed0e602" style="width: 220px; margin-top: 5px;"/>
+                      </div>
+                      <div>
+                        <img src="https://firebasestorage.googleapis.com/v0/b/eciis-splab.appspot.com/o/admin_email%2Fbr.png?alt=media&token=22012e23-60ad-4e63-969c-76e9000343f3" style="width: 150px; margin-top: 10px;"/>
+                      </div>
+                      <div style="margin-top: 23px; font-size: 10px;">
+
```
Comment (mayzabeel) [inline backend/templates/invite_user.html]: rlb

File: backend/templates/invite_user.html [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+                                    <span style="font-size: 7px;">{{inst_email}}</span>
+                                  </div>
+                                  <div style="height: 10px;">
+                                    <span style="font-size: 7px;">Convidado por: {{inviter}}</span>
+                                  </div>
+                                </div>
+                              </div>
+                            </div>
+                            <div style="margin-top: 10px;" >
+                              <button style="background-color: #009688; width: 90%; height: 25px; border: none;"><span style="text-align: center; font-size: 8px; color: #FFFFFF">ACEITAR CONVITE</span></button>
```
Comment (mayzabeel) [inline backend/templates/invite_user.html]: direcionar para página de login

File: backend/templates/invite_user.html [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+                      <b>CONHEÇA O CIS</b>
+                    </div>
+                    <div>
+                      <img src="https://firebasestorage.googleapis.com/v0/b/eciis-splab.appspot.com/o/admin_email%2Fimg2.ico?alt=media&token=93638323-15cd-4b46-abf7-1a6ee491ab64" style="width: 100px; margin-top: 10px;"/>
+                    </div>
+                    <div style="height: 190px;">
+                      <p style="font-size: 18px; text-align: center; color: #BDBDBD; width: 90%;"><b>Plataforma Virtual do Complexo Indústrial da Saúde</b></p>
+                      <p style="font-size: 8px; color: black;">A Plataforma Virtual do Complexo Industrial da Saúde (e-CIS) é uma ferramenta criada para favorecer a interlocução, articulação, discussão, proposições de medidas e ações sobre o Complexo Industrial da Saúde (CIS).</p>
+                      <div style="height: 25px; margin-top: 10px;" align="center" valign="bottom">
+                        <button style="background-color: #009688; width: 90%; height: 100%; border: none;"><span style="text-align: center; font-size: 8px; color: #FFFFFF">CONFIRA AS FUNCIONALIDADES</span></button>
```
Comment (mayzabeel) [inline backend/templates/invite_user.html]: direcionar para landing-page

File: backend/worker.yaml [hunk 5]
```diff
@@ -3,6 +3,10 @@ api_version: 1
 threadsafe: true
 service: worker
 
+libraries:
```
Comment (andreldsa) [inline backend/worker.yaml]: You don't need this property anymore, once you added to the requirements files. But, be aware that you have to execute the pip install before to execute the application.

File: backend/send_email_hierarchy/email_sender.py [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+            target='worker',
+            queue_name='notifications',
+            params={
+                'invitee': self.receiver,
+                'subject': self.subject,
+                'html': self.html,
+                'json': json.dumps(email_json if email_json else self.body)
+            }
+        )
+
```
Comment (andreldsa) [inline backend/send_email_hierarchy/email_sender.py]: RLB

File: backend/worker.yaml (hunk 5 above)
Comment (raonismaneoto) [reply backend/worker.yaml]: Nice

**Pull request merged**